routes/employees.js

```diff
@@ -1,17 +1,20 @@
 const express = require('express')
 const router = express.Router()
 const employeesController = require('../controllers/employees')
+const createEmployeeController = require('../controllers/employees/create')
+const updateEmployeeController = require('../controllers/employees/update')
+const deleteEmployeeController = require('../controllers/employees/delete')
 
 router.get('/', employeesController.getEmployees)
 
 router.get('/:id', employeesController.getEmployeeById)
 
 router.get('/firstname/:first_name', employeesController.getEmployeeByFirstName)
```

Comment:
Not sure why we need this route or function. Should only need to get by id and get all. Can you check with Michael and Destiny and clarify if this functionality is needed?